docs/documentation/hardpy_panel.md

```diff
@@ -57,6 +57,24 @@ Contains an instruction or question, a number input field, and a `confirm` butto
 
 ![num_input_dialog_box](../img/dialog_box/num_input_dialog_box.jpg)
 
+#### radiobutton
+
+Contains radiobutton widget.
+
+* The user selects one option from several possible ones.
+* Returns the contents of the selected item as a string.
+
+![radiobutton_dialog_box](../img/dialog_box/radiobutton_dialog_box.jpg)
```

Comment:
Incorrect image format. Image must be .`jpg`, not `.png`

Reply:
Ready